simphony/io/cuds_file.py

```diff
@@ -179,3 +261,23 @@ def iter_particle_containers(self, names=None):
             names = self._particle_containers.keys()
         for name in names:
             yield self.get_particle_container(name), name
+
+    def iter_meshes(self, names=None):
+        """Returns an iterator over a subset or all
+        of the meshes. The iterator iterator yields
+        (name, mesh) tuples for each mesh container
+        contained in the file.
+
+        Parameters
+        ----------
+        names : list of str
+            names of specific mesh containers to be iterated over.
+            If names is not given, then all meshes will be iterated over.
+
+        """
+
+        names = copy.deepcopy(names)
```

Comment:
no need to deep copy, this line is not necessary since you are not mutating the list of names.